### ISSUE TITLE
High Sierra upgrade feedback

### ISSUE THREAD

**Issue by clounie** [ForkIssues/Tracker#122]
General information on my experience upgrading to High Sierra...nothing really bad, but (especially regarding A) some improvements to be made?

## A. Repository Tabs
Don't know if this was expected or not, but Fork lost all my repository tabs. However, once I re-opened the repository, it remembered my pinned branches (very awesome 🥇 )

-------------
## B. License fun
Fork dialog got stuck (and I had to force quit) when I followed the following steps:
1) Said to run XCode to accept license
2) Accepted XCode license
3) Got bored and decided to use Fork local copy
4) Selected Fork local copy

Once I force quit and restarted Fork, the local copy was selected - but until I force quit, none of the buttons would do anything.

**Comment by DanPristupov** [comment]
@clounie thank you for the feedback.

This migration has happened because of my fault. I did not migrate 100% of users to Fork's git instance in time and had to create a workaround. I hope this is the last such a migration for Fork.

Regarding the particular problems:
A. Yes, Fork forgets the opened tabs. This is a known behaviour, but I think it's ok, since it happens only once.
B. I noticed a few-second lag when I had accepted the XCode license, but then everything worked out problem.
The thing is that for me such a migration is very difficult to test. I had to use a clean machine without an ability to repeat the xcode update and also without debug (because XCode is not identified :) ).

In the future I'm going to create a clean OS X VM which will allow me to create snapshots and roll back the changes when I need.
